portal/templates/utilisation-reporting-service/utilisation-report-upload/utilisation-report-upload.njk

```diff
@@ -0,0 +1,66 @@
+{% extends "utilisation-reporting-service/index.njk" %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% block pageTitle %}Utilisation Report Upload{% endblock %}
+{% block content %}
+    {% if errorSummary %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: errorSummary,
+          attributes: {
+            'data-cy': 'error-summary'
+          }
+        }) }}
+    {% endif %}
+    <h1 class="govuk-heading-l" data-cy="main-heading">
+        General Export Facility reporting
+    </h1>
+    <h2 class="govuk-heading-m" data-cy="sub-heading">
+        June 2023 status report is due
+    </h2>
+    <div class="govuk-inset-text">
+        All issued facilities must be updated and sent to UKEF by 14 July 2023.
+    </div>
+    <p class="govuk-body">For each issued facility you must provide:</p>
+    <ul class="govuk-list govuk-list--bullet">
+        <li>facility utilisation accurate for the report month-end date</li>
+        <li>the total fees accrued</li>
+        <li>the fee payable to UKEF</li>
+    </ul>
+    <p class="govuk-body">If there has been no movement against a facility you must still report this.</p>
+    <form method="POST" data-cy="form" enctype="multipart/form-data">
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
```

Comment:
Ah I can delete this I think because it's in the formaction (multipart/form-data forms mean that this input doesn't get pushed through so we have to rely on the csrf token as a query parameter in the header instead)